pkg/fission-cli/cmd/environment/create.go

```diff
@@ -119,7 +119,14 @@ func createEnvironmentFromCmd(input cli.Input) (*fv1.Environment, error) {
 	}
 
 	poolsize := input.Int(flagkey.EnvPoolsize)
+
 	if input.IsSet(flagkey.EnvPoolsize) {
+		if input.Int(flagkey.EnvPoolsize) > 0 {
+			poolsize = input.Int(flagkey.EnvPoolsize)
```

Comment:
Can you make the logic more concise because if the flag is not set you will get default value 3, at line 121